pkg/registry/servicecatalog/instance/strategy_test.go

```diff
@@ -63,15 +74,19 @@ func TestInstanceUpdate(t *testing.T) {
 	}{
 		{
 			name:  "no spec change",
-			older: instanceWithOldSpec(),
-			newer: instanceWithOldSpec(),
+			older: getTestInstance(),
+			newer: getTestInstance(),
 		},
-		//{
-		//	name:  "spec change",
-		//	older: instanceWithOldSpec(),
-		//	newer: instanceWithNewSpec(),
-		//	shouldGenerationIncrement: true,
-		//},
+		//		{
```

Comment:
old?  delete?

Reply:
Those tests will be uncommented soon when we support updates. There is a TODO comment in the file regarding that.